### ISSUE TITLE
Big seqmonk project files cause NumberFormatException

### ISSUE THREAD

**Issue by s-andrews** [s-andrews/SeqMonk#212]
We get occasional bug reports resulting from project files which have been corrupted during copying or downloading and generate all kinds of odd parser errors when reloaded.  Recently though we've had a few very similar reports which look like these, but are too similar to each other to be a coincidence.  These probably represent an actual underlying bug somewhere.  The crash report profile is:

```
SeqMonk Version:1.45.4

Operating System:Linux - 3.10.0-514.26.2.el7.x86_64

Java Version:11.0.2 - 11.0.2+9

java.lang.NumberFormatException
For input string: "2233971288"

java.base/java.lang.NumberFormatException.forInputString(Unknown Source) java.base/java.lang.Integer.parseInt(Unknown Source) java.base/java.lang.Integer.parseInt(Unknown Source)
uk.ac.babraham.SeqMonk.DataParsers.SeqMonkParser.parseSamples(SeqMonkParser.java:630)
uk.ac.babraham.SeqMonk.DataParsers.SeqMonkParser.run(SeqMonkParser.java:223)
java.base/java.lang.Thread.run(Unknown Source)

```

**Comment by s-andrews** [comment]
I think I've figured out what's going on.  A while back we changed the total read count counter from an int to a long since we were ending up with samples with more than 2^31 reads in them.  This naturally passed through to the data writer because the return type for getTotalReadCount changed.  However, the parser was never updated, so if you had a dataset where the extra capacity was used then it would fail to parse when re-opened.
